website/docs/feature-flag-tutorials/golang/golang-examples.md

```diff
@@ -0,0 +1,361 @@
+---
+title: Go Feature Flag Examples
+slug: /feature-flag-tutorials/golang/examples
+---
+
+In our [Go feature flag tutorial](/feature-flag-tutorials/golang), we implemented a simple feature flag that could be turned on and off. This document will walk you through some common examples of using feature flags in Go.
+
+We built many features into Unleash, our open-source feature flag platform, to address the complexities of releasing code. This tutorial will explore the following:
+
+- [Gradual Rollouts for Go Apps](#gradual-rollouts-for-go-apps)
+- [Canary Deployments in Go](#canary-deployments-in-go)
+  - [What is a canary deployment?](#what-is-a-canary-deployment)
+  - [Why use canary deployments?](#why-use-canary-deployments)
+  - [How to do canary deployments with a feature flag in Go?](#how-to-do-canary-deployments-with-a-feature-flag-in-go)
+  - [Configure strategy constraints for canary deployments](#configure-strategy-constraints-for-canary-deployments)
+- [Server-side A/B Testing in Go](#server-side-ab-testing-in-go)
+- [Feature Flag Analytics and Reporting in Go](#feature-flag-analytics-and-reporting-in-go)
+  - [Enable impression data events in Go](#enable-impression-data-events-in-go)
+- [Application Metrics and Monitoring for Go apps](#application-metrics-and-monitoring-for-go-apps)
+- [Feature Flag Audit Logs in Go](#feature-flag-audit-logs-in-go)
+- [Flag Automation and Workflow Integration for Go Apps](#flag-automation-and-workflow-integration-for-go-apps)
+
+## Gradual Rollouts for Go Apps
+
+It is common to use feature flags to roll out changes to a percentage of users, and we can use Unleash to do that too.
+
+Navigate to the gradual rollout page where you can edit your strategy.
+
+![The "edit strategy" button uses a pencil icon and is located on every strategy.](/img/react-ex-grad-rollout-edit.png)
+
+Adjust the percentage of users to 50% or whichever value you choose, and refresh your app in the browser to see if your user has the new feature experience. This might take 30 seconds for a server-side app to propagate.
```

Comment:
I don't get this 🤔 why would a 50% rollout guarantee that my user has the new experience?